tests/t281_agent_trace_toggle.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+import subprocess as sp
+from time import sleep
+
+from runtest import TestBase
+
+class TestCase(TestBase):
```

Comment:
It would be nice if you add some comment what this test expects.  IIUC the program calls the `func()` three times but the trace disabled for the second one so the result should be 2.